app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,199 @@
+@import url('https://fonts.googleapis.com/css?family=Proxima+Nova');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+  background-color: #dedede;
+  font-family: 'proxima-nova', sans-serif;
+}
+
+.splash-screen {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 100vh;
+  gap: 10em;
+}
+
+.splash-screen h1 {
+  font-size: 2.5em;
+  font-weight: 800;
+  font-style: italic;
+}
+
+.splash-screen div {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.splash-screen .login-btn,
+.splash-screen .signup-btn {
+  width: 80%;
+  height: 10vh;
+  margin: 0 auto;
+  background-color: #3778c2;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  font-weight: 900;
+  font-size: 1.5em;
+}
+
+.splash-screen .signup-btn {
+  background-color: #dedede;
+  color: #acacac;
+}
+
+a {
+  text-decoration: none;
+}
+
+.form-container nav {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 2%;
+  align-items: center;
+  background-color: #3778c2;
+  margin-bottom: 5%;
+}
+
+.form-container nav .arrow-link,
+.form-container nav h2,
+.form-container nav input {
+  color: #fff;
+  font-size: 1.2em;
+  font-weight: 800;
+}
+
+.form-container nav input {
+  border: none;
+  background-color: transparent;
+}
+
+.form-container .field input {
+  width: 100%;
+  height: 10vh;
+  padding: 2%;
+  outline: none;
+  margin-bottom: 1%;
+  font-size: 1.5em;
+  font-weight: 800;
+}
+
+.form-container .forget {
+  width: 100%;
+  text-align: center;
+  margin-top: 2%;
+}
```

Comment:
Your is working fine, good job! 👏 However, it will be a good idea if we can add a few tweaks to the design aspect so as to improve the look and feel. Please take note that it's one of the project's objectives that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's improve upon the design aspect by introducing the following tweaks:

- [ ] Let's ensure the Category form matches the original design.
- [ ] Similarly, the form for adding transactions should match the original design.
- [ ] The blue header at the top should appear at on the form pages, and should have a back arrow icon to navigate to previous page, in case the user decides to navigate back without filling the form.

**Category form:**        |  **Transaction form:**        |  **Expected:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/CaptainBawa/Budget-App/assets/12745474/b0c5b79f-de1d-48d8-89f0-875e0cb5c3de"> | <img width="800" alt="Home Page" src="https://github.com/CaptainBawa/Budget-App/assets/12745474/94e6944d-d0cb-42b8-9c44-5f4005c4ac8c"> | <img width="800" alt="Home Page" src="https://github.com/CaptainBawa/Budget-App/assets/12745474/7a6a0a64-afb3-48af-9893-c20a7872e643">